src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -672,6 +670,12 @@ public string ApplyGetLogging()
             return _expressionAnonymous;
         }
 
+        internal string ApplyGetLogging(ISanitizedNameProvider nameProvider = null)

```

Comment:
Which overload does just calling `ApplyGetLogging()` bind against?  Line 661, or line 673 with nameProvider=null?

Reply:
Having 2 separate methods is import from a breaking change perspective, but the trip up is the default value for the param.  Basically:
- should we remove the default value for the param?
- should the existing call in ApplyGetLogging() just become a call to ApplyGetLogging(null).